website/addons/wiki/templates/edit.mako

```diff
@@ -354,42 +353,39 @@
 </div>
 
 <%def name="javascript_bottom()">
-<% import json %>
 ${parent.javascript_bottom()}
 <script>
 
     var canEditBody = ${user['can_edit_wiki_body'] | sjson, n};
     var isContributor = ${user['can_edit']  | sjson, n};
 
-    var canEditPageName = isContributor && ${json.dumps(
-        wiki_id and wiki_name != 'home'
-    )};
+    var canEditPageName = isContributor && ${(wiki_id and wiki_name != 'home') | sjson, n };
 
     window.contextVars = window.contextVars || {};
     window.contextVars.wiki = {
         canEdit: canEditBody,
         canEditPageName: canEditPageName,
-        usePythonRender: ${json.dumps(use_python_render)},
```

Comment:
`json.dumps` was deprecated in favor of the `sjson, n` pair. See JIRA ticket for link to conversion guidelines.

Also see function definition for a brief description of vectors the newer filter is designed to handle. The `n` part of the filter pair ensures that we do not apply HTML-specific rules to a variable that is not HTML.
https://github.com/CenterForOpenScience/osf.io/blob/develop/website/util/sanitize.py#L121